### PR TITLE
Fix validation when adding a new repository to cosmos.

### PR DESCRIPTION
**Pull request by rcorral**
refs: https://dcosjira.atlassian.net/browse/DCOS-289

### CODE REVIEW

**Comment by mlunoe** [comment]
Thanks for fixing! Works great 👍 ⛵

File: src/js/components/modals/AddRepositoryFormModal.js [hunk 1]
```diff
@@ -60,6 +60,11 @@ class AddRepositoryFormModal extends mixin(StoreMixin) {
 
   getWithinRangeFunction(min, max) {
```
Comment (orlandohohmeier) [inline src/js/components/modals/AddRepositoryFormModal.js]: How about using the shine new [`isNumberInRange`](https://github.com/dcos/dcos-ui/blob/master/src/js/utils/ValidatorUtil.js#L48) in combination with [`!isDefined`](https://github.com/dcos/dcos-ui/blob/master/src/js/utils/ValidatorUtil.js#L2) instead? That would make it way easier to under the validation rule.

Comment (orlandohohmeier) [inline src/js/components/modals/AddRepositoryFormModal.js]: ✔️  Adjusted

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by rcorral** [comment]
Thanks @orlandohohmeier!

**Pull request merged**